### PR TITLE
Rubocop: Style/GuardClause

### PR DESCRIPTION
**Pull request by jaredmoody**
Most of the cops left are going to require some refactoring or at least manual changes, so I’ll probably open PRs to discuss before making the changes.

I like this guard clause style, what do you guys think?

### CODE REVIEW

File: benchmark/images.rb [hunk 1]
```diff
@@ -20,11 +20,10 @@
 LARGE_FILE = File.expand_path(File.join(__FILE__, "..", "large.jpg"))
 
 def check_file(file, url)
-  unless File.exist?(file)
-    puts "Downloading #{file}..."
-    uri = URI.parse(url)
-    File.open(file, "w") { |f| f.write(Net::HTTP.get_response(uri).body) }
-  end
+  return if File.exist?(file)
```
Comment (PragTob) [inline benchmark/images.rb]: big fan of an empty line after a guard statement!

File: tasks/changelog.rb [hunk 2]
```diff
@@ -81,10 +81,9 @@ def misc_changes(commit_range, categorized_commits)
   end
 
   def changes_under_heading(title, commits)
-    if commits.any?
-      heading = heading(title, commits.length)
-      heading << commits.join("\n")
-    end
+    return if commits.empty?
```
Comment (PragTob) [inline tasks/changelog.rb]: empty line sort of thing :)

**Comment by PragTob** [comment]
also tests are failing but I have no idea why...

**Comment by jaredmoody** [comment]
OK, this could definitely use a second set of eyes to just to make sure I didn't miss anything, but finished adding guard clauses everywhere and build is now passing.

**Comment by jasonrclark** [comment]
Looks like merges on `.rubocop_todo.yml` landed you a merge conflict 😭 

Tested out the samples and they all seem happy to me, though, so once we get the conflict resolved I think we'll be 👌 to proceed.

**Comment by jaredmoody** [comment]
Conflict resolved!

**Comment by jasonrclark** [comment]
💥

**Pull request merged**